app/src/main/java/com/rockthevote/grommet/data/db/dao/RegistrationDao.kt

```diff
@@ -12,9 +12,9 @@ interface RegistrationDao {
     fun getAll(): List<Registration>
 
     @Insert
-    fun insert(vararg registration: Registration)
+    fun insert(vararg registrations: Registration)
 
     @Delete
-    fun delete(registration: Registration)
+    fun delete(vararg registrations: Registration)
```

Comment:
this will only work if the registration object matches 100% i believe, meaning you can't just pass it a row ID. Not saying don't use it, just an FYI

Reply:
Whoa, interesting. I thought it used the primary key. I'll look deeper into that. Either way it shouldn't matter here for what I'm using it for. I pull the `Registration`s direct from the DB, and then use the same list to determine which needs to be deleted.

Reply:
From testing, at least for my case, this is working.

Reply:
hm, well alrighty then 👍